### PR TITLE
added tags

### LINKED ISSUE #2: Tags
Hi all,
Great work with this module!
Could you add defined_tags and freeform_tags ?

### PR DESCRIPTION
**Pull request by nugoncal**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -73,6 +73,10 @@ variable "mysql_db_system_data_storage_size_in_gb" {
   description = "Initial size of the data volume in GiBs that will be created and attached."
 }
 
+variable "mysql_db_system_defined_tags" {
+  default = {}
+}
```
Comment (dralquinta) [inline variables.tf]: Description should be added for documentation purposes

File: variables.tf [hunk 2]
```diff
@@ -85,6 +89,10 @@ variable "mysql_db_system_fault_domain" {
   description = "The fault domain on which to deploy the Read/Write endpoint. This defines the preferred primary instance."
 }
 
+variable "mysql_db_system_freeform_tags" {
+  default = {}
+}
```
Comment (dralquinta) [inline variables.tf]: Description should be added for documentation purposes

**Review by dralquinta — CHANGES_REQUESTED** [review]
3 minor changes for approving merge 

- resolve the 2 comments on variables.tf
- update README.md with the extra variables. You can use terraform-docs to generate the markdown by executing terraform-docs markdown . And then replacing the current content by the one outputted there. 

Terraform docs is available here for usage. https://github.com/terraform-docs/terraform-docs

Thanks.

**Comment by nugoncal** [comment]
Added description to the variables and the README entries, as requested.
The text chosen is the official one present in HashiCorp Registry.
README preview: https://github.com/nugoncal/terraform-oci-cloudbricks-mysql-database/tree/tag

**Review by dralquinta — APPROVED** [review]
LGTM

**Comment by dralquinta** [comment]
All set. Approved and merged. I'll do the release sometime today. Thanks for the work!

**Pull request merged**